### ISSUE TITLE
Errors in .d.ts files when running webpack

### ISSUE THREAD

**Issue by sharmad-nachnolkar** [TypeStrong/fork-ts-checker-webpack-plugin#96]
Have moved from type checking via ts-loader to this plugin.

However many of the .d.ts files have started showing errors which were not reported earlier by ts-loader. Attaching webpack config and tsconfig

webpack config - 
```
       {
                test: /\.tsx?$/,
                loader: 'ts-loader',
                options: {
                    "transpileOnly": true
                  },
        }
```

ts config -
```
{
    "compilerOptions": {
        "module": "commonjs",
        "noImplicitAny": false,
        "removeComments": true,
        "preserveConstEnums": true,
        "jsx": "react",
        "target": "es5",
        "listFiles": true,
        "sourceMap": false,
        "allowUnreachableCode": true,
        "noImplicitUseStrict": true,
        "allowSyntheticDefaultImports": true,
        "experimentalDecorators": true,
        "lib": [
            "dom",
            "es5",
            "es2015",
            "es2015.promise"
        ]
    },
    "transpileOnly": true,
    "reportFiles": ["*.tsx"],   
    "isolatedModules": true, 
    "include": [
    ],
    "exclude": [

    ]
}
```

**Comment by piotr-oles** [comment]
Did you try "skipLibCheck" option in tsconfig?

03.02.2018 3:05 PM "sharmad-nachnolkar" <notifications@github.com>
napisał(a):

**Comment by johnnyreilly** [comment]
Just to add to @piotr-oles comments, ts-loader enables `skipLibCheck` by default. I don't believe fork-ts-checker-webpack-plugin does. It might be nice to align the defaults between the two perhaps. Would avoid confusion for people when they make the switch to using fork-ts-checker-webpack-plugin

**Comment by sharmad-nachnolkar** [comment]
@piotr-oles  @johnnyreilly  Thanks a lot. Setting `skipLibCheck` to `true` did the trick.
Yes, I think as @johnnyreilly  mentioned, it would be nice to align the defaults of ts-loader and ts-checker-webpack-plugin

**Comment by guilhermesimoes** [comment]
Hello 👋 
Was just wondering why this never came true:
> it would be nice to align the defaults of ts-loader and ts-checker-webpack-plugin

It's a bit unexpected that this plugin has a different default (`skipLibCheck: true`) from [TS ](https://www.typescriptlang.org/tsconfig#skipLibCheck)(`skipLibCheck: false`).
